### ISSUE TITLE
Want to reply to all message

### ISSUE THREAD

**Issue by wasimjee** [IwraStudios/Whatsapp-Bot#18]
Hi,

I am trying to reply to all new messages.

and for this, I am using the code

`Task.Run(() => _driver.MessageScanner());`

But not reading any new message.

it seems the system is unable to find the class
`unread-count`

please guide me where am I wrong.

**Comment by IwraStudios** [comment]
That is correct, WhatsApp changed their css classes, they will need to be updated before this library is fully functional again

**Comment by k4m4r82** [comment]
So how to check WhatsApp css classes ?

**Comment by IwraStudios** [comment]
@k4m4r82 you can use developers tools that is included in almost all big browsers (just go to web.whatsapp.com)

For the new `unread-count` you want to find a class that is in the element which contains the 'new message counter', also commonly called a badge notification

**Comment by k4m4r82** [comment]
@IwraStudios: Ok, thank you for the information :-)
